connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/network/UplynkAd.kt

```diff
@@ -73,7 +74,7 @@ data class UplynkAd(
      * Contains the custom set of VAST extensions returned by the ad server.
      * Each custom extension is reported as an object.
      */
-    val extensions: List<Map<String, String>>? = null,
+    val extensions: JsonElement? = null,
```

Comment:
Also add documentation about why this is a `JsonElement`, like you wrote for `UplynkPlayedAd.extensions`.